integration/simple_test.go

```diff
@@ -1106,6 +1106,95 @@ func (s *SimpleSuite) TestMirrorWithBody() {
 	assert.Equal(s.T(), int32(0), val2)
 }
 
+func (s *SimpleSuite) TestMirrorWithoutBody() {
```

Comment:
I thin it is better to rename and adapt TestMirrorWithBody, to check in addition the new router without body, could you please merge the two tests?

Reply:
I merged the tests, let me know if there is something else to do.
I've not renamed the test, since I did not came up with a better name (my modifications also relate to body).

Reply:
Indeed, this is ok with the same name, thanks for the rework!